components/layouts/TheMenu/MobileMenu.vue

```diff
@@ -5,52 +5,83 @@
     >
       <template v-for="link in mainLevelLinks">
         <AppLink
+          v-if="!link.sublinks"
           :key="link.url"
-          class="mobile-menu__link"
+          class="mobile-menu__entry"
           :class="{
-            'mobile-menu__link_active': isActive(link),
-            'mobile-menu__link_is-parent': isParent(link)
+            'mobile-menu__entry_active': isActive(link),
+            'mobile-menu__entry_is-parent': isParent(link)
           }"
           v-bind="appLinkFromNavLink(link)"
           kind="secondary"
         >
-          <p class="mobile-menu__link-label">
+          <p class="mobile-menu__entry-label">
             {{ link.label }}
           </p>
         </AppLink>
-        <AppLink
-          v-for="sublink in getSubLinks(link)"
-          :key="`sublink:${sublink.url}`"
-          class="
-          mobile-menu__link
-          mobile-menu__link_second-level
-        "
-          :class="{ 'mobile-menu__link_active': isActive(sublink) }"
-          v-bind="appLinkFromNavLink(sublink)"
-          kind="secondary"
+        <cv-dropdown
+          v-else
+          ref="communityDropdown"
+          :key="link.url"
+          class="mobile-menu__entry"
+          :class="{ 'mobile-menu__entry_active': isCommunityActive() }"
+          placeholder="Community"
         >
-          <p class="mobile-menu__link-label">
-            {{ sublink.label }}
-          </p>
-        </AppLink>
+          <li v-for="sublink in getSubLinks(link)" :key="`sublink:${sublink.url}`">
+            <AppLink
+              class=" mobile-menu__entry mobile-menu__entry_second-level"
+              :class="{ 'mobile-menu__entry_active': isActive(sublink) }"
+              v-bind="appLinkFromNavLink(sublink)"
+              kind="secondary"
+            >
+              <p class="mobile-menu__entry-label">
+                {{ sublink.label }}
+              </p>
+            </AppLink>
+          </li>
+        </cv-dropdown>
       </template>
     </nav>
     <footer class="mobile-menu__footer">
       <div class="mobile-menu__footer-inner-container">
-        <p class="mobile-menu__footer-text">
-          ©Qiskit | All Rights Reserverd
-        </p>
+        <FooterSection class="mobile-menu__footer-contact" v-bind="stayConnectedElements" icons-only :theme="theme" />
       </div>
+      <p class="mobile-menu__footer-text">
+        ©Qiskit | All Rights Reserved
+      </p>
     </footer>
   </section>
 </template>
 
 <script lang="ts">
-import { Component, Mixins } from 'vue-property-decorator'
+import { Component, Mixins, Prop, Watch } from 'vue-property-decorator'
 import MenuMixin from '~/mixins/menu'
 
+import {
+  STAY_CONNECTED_LINKS
+} from '~/constants/menuLinks'
+
 @Component
-export default class MobileMenu extends Mixins(MenuMixin) {}
+export default class MobileMenu extends Mixins(MenuMixin) {
+  @Prop({ type: Boolean, default: false })
```

Comment:
Usually, we write the property name following the decorator. Take a look at other components to see how.